client/src/rpc_client.rs

```diff
@@ -843,25 +880,36 @@ impl RpcClient {
         )
     }
 
+    fn get_config_for_program_accounts(&self) -> RpcProgramAccountsConfig {
```

Comment:
```suggestion
    fn make_program_accounts_config(&self) -> RpcProgramAccountsConfig {
```
`get` typically implies a call or request; wdyt about this alternative?

Reply:
just trying to share code an eliminate duplication

Reply:
Yep, just nitpicking the method name :)